app/src/components/InputElement/InputElement.styles.js

```diff
@@ -3,12 +3,35 @@ import background from "../../images/bg-shorten-desktop.svg";
 
 export const StyledForm = styled.form`
   position: absolute;
```

Comment:
I haven't seen that before but this form does not have to be positioned as absolute. This is not correct in my opinion. I see no reason to make it absolute

Reply:
Oh ok, it's because this form has to be kind of between two sections. I forgot about this, sorry 👍